This patch when applied allows the PostScript files in the
tcl7.5-tk4.1-man-2ps-us.tar.gz archive to be printed on either
a LETTER or A4 printer.

Cary D. Renzema
caryr@mxim.com
May 3rd, 1996


diff -c orig/README new/README
*** orig/README	Mon Apr 15 12:12:52 1996
--- new/README	Thu May  2 09:32:53 1996
***************
*** 1,6 ****
  These PostScript files are the manual pages and table of contents for
! tcl7.5 and tk4.1. They should be printed on a DOUBLE sided, LETTER
! size printer. All manual pages start on an odd page. All the files
  have an even number of pages. Some time in the future, when I get a
  spare moment, I am going to release the code that build the PostScript
  from the manual pages, but for now the PostScript will have to do.
--- 1,6 ----
  These PostScript files are the manual pages and table of contents for
! tcl7.5 and tk4.1. They should be printed on a DOUBLE sided, LETTER or
! A4 size printer. All manual pages start on an odd page. All the files
  have an even number of pages. Some time in the future, when I get a
  spare moment, I am going to release the code that build the PostScript
  from the manual pages, but for now the PostScript will have to do.
diff -c orig/tcl-c-man.ps new/tcl-c-man.ps
*** orig/tcl-c-man.ps	Wed Apr 24 14:26:11 1996
--- new/tcl-c-man.ps	Thu May  2 09:25:44 1996
***************
*** 1,6 ****
  %!PS-Adobe-3.0
  %%Creator: groff version 1.10
! %%CreationDate: Wed Apr 24 14:26:09 1996
  %%DocumentNeededResources: font Times-Roman
  %%+ font Times-Bold
  %%+ font Times-Italic
--- 1,6 ----
  %!PS-Adobe-3.0
  %%Creator: groff version 1.10
! %%CreationDate: Thu May  2 09:25:43 1996
  %%DocumentNeededResources: font Times-Roman
  %%+ font Times-Bold
  %%+ font Times-Italic
***************
*** 175,181 ****
  %%IncludeResource: font Times-Italic
  %%IncludeResource: font Symbol
  grops begin/DEFS 1 dict def DEFS begin/u{.001 mul}bind def end/RES 72
! def/PL 792 def/LS false def/ENC0[/asciicircum/asciitilde/Scaron/Zcaron
  /scaron/zcaron/Ydieresis/trademark/quotesingle/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
--- 175,181 ----
  %%IncludeResource: font Times-Italic
  %%IncludeResource: font Symbol
  grops begin/DEFS 1 dict def DEFS begin/u{.001 mul}bind def end/RES 72
! def/PL PLG def/LS false def/ENC0[/asciicircum/asciitilde/Scaron/Zcaron
  /scaron/zcaron/Ydieresis/trademark/quotesingle/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
diff -c orig/tcl-c-toc.ps new/tcl-c-toc.ps
*** orig/tcl-c-toc.ps	Wed Apr 24 14:26:19 1996
--- new/tcl-c-toc.ps	Thu May  2 09:25:54 1996
***************
*** 1,6 ****
  %!PS-Adobe-3.0
  %%Creator: groff version 1.10
! %%CreationDate: Wed Apr 24 14:26:19 1996
  %%DocumentNeededResources: font Times-Roman
  %%+ font Times-Bold
  %%DocumentSuppliedResources: procset grops 1.10 0
--- 1,6 ----
  %!PS-Adobe-3.0
  %%Creator: groff version 1.10
! %%CreationDate: Thu May  2 09:25:54 1996
  %%DocumentNeededResources: font Times-Roman
  %%+ font Times-Bold
  %%DocumentSuppliedResources: procset grops 1.10 0
***************
*** 171,177 ****
  %%IncludeResource: font Times-Roman
  %%IncludeResource: font Times-Bold
  grops begin/DEFS 1 dict def DEFS begin/u{.001 mul}bind def end/RES 72
! def/PL 792 def/LS false def/ENC0[/asciicircum/asciitilde/Scaron/Zcaron
  /scaron/zcaron/Ydieresis/trademark/quotesingle/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
--- 171,177 ----
  %%IncludeResource: font Times-Roman
  %%IncludeResource: font Times-Bold
  grops begin/DEFS 1 dict def DEFS begin/u{.001 mul}bind def end/RES 72
! def/PL PLG def/LS false def/ENC0[/asciicircum/asciitilde/Scaron/Zcaron
  /scaron/zcaron/Ydieresis/trademark/quotesingle/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
diff -c orig/tcl-man.ps new/tcl-man.ps
*** orig/tcl-man.ps	Wed Apr 24 14:25:18 1996
--- new/tcl-man.ps	Thu May  2 09:24:50 1996
***************
*** 1,6 ****
  %!PS-Adobe-3.0
  %%Creator: groff version 1.10
! %%CreationDate: Wed Apr 24 14:25:17 1996
  %%DocumentNeededResources: font Times-Roman
  %%+ font Times-Bold
  %%+ font Times-Italic
--- 1,6 ----
  %!PS-Adobe-3.0
  %%Creator: groff version 1.10
! %%CreationDate: Thu May  2 09:24:48 1996
  %%DocumentNeededResources: font Times-Roman
  %%+ font Times-Bold
  %%+ font Times-Italic
***************
*** 175,181 ****
  %%IncludeResource: font Times-Italic
  %%IncludeResource: font Symbol
  grops begin/DEFS 1 dict def DEFS begin/u{.001 mul}bind def end/RES 72
! def/PL 792 def/LS false def/ENC0[/asciicircum/asciitilde/Scaron/Zcaron
  /scaron/zcaron/Ydieresis/trademark/quotesingle/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
--- 175,181 ----
  %%IncludeResource: font Times-Italic
  %%IncludeResource: font Symbol
  grops begin/DEFS 1 dict def DEFS begin/u{.001 mul}bind def end/RES 72
! def/PL PLG def/LS false def/ENC0[/asciicircum/asciitilde/Scaron/Zcaron
  /scaron/zcaron/Ydieresis/trademark/quotesingle/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
diff -c orig/tcl-toc.ps new/tcl-toc.ps
*** orig/tcl-toc.ps	Wed Apr 24 14:25:20 1996
--- new/tcl-toc.ps	Thu May  2 09:24:52 1996
***************
*** 1,6 ****
  %!PS-Adobe-3.0
  %%Creator: groff version 1.10
! %%CreationDate: Wed Apr 24 14:25:20 1996
  %%DocumentNeededResources: font Times-Roman
  %%DocumentSuppliedResources: procset grops 1.10 0
  %%Pages: 4
--- 1,6 ----
  %!PS-Adobe-3.0
  %%Creator: groff version 1.10
! %%CreationDate: Thu May  2 09:24:52 1996
  %%DocumentNeededResources: font Times-Roman
  %%DocumentSuppliedResources: procset grops 1.10 0
  %%Pages: 4
***************
*** 169,175 ****
  %%BeginSetup
  %%IncludeResource: font Times-Roman
  grops begin/DEFS 1 dict def DEFS begin/u{.001 mul}bind def end/RES 72
! def/PL 792 def/LS false def/ENC0[/asciicircum/asciitilde/Scaron/Zcaron
  /scaron/zcaron/Ydieresis/trademark/quotesingle/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
--- 169,175 ----
  %%BeginSetup
  %%IncludeResource: font Times-Roman
  grops begin/DEFS 1 dict def DEFS begin/u{.001 mul}bind def end/RES 72
! def/PL PLG def/LS false def/ENC0[/asciicircum/asciitilde/Scaron/Zcaron
  /scaron/zcaron/Ydieresis/trademark/quotesingle/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
  /.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef/.notdef
